README.md

```diff
@@ -916,6 +925,7 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
 
 [[crypto](https://crates.io/keywords/crypto), [cryptography](https://crates.io/keywords/cryptography)]
 
+* [arkworks-rs](https://github.com/arkworks-rs) - arkworks is a Rust ecosystem for zkSNARK programming.
```

Comment:
This will need adding to https://github.com/rust-unofficial/awesome-rust/blob/main/src/main.rs#L66 as it's an organisation, and we don't have star counts for those. The organisation is allowed because it's got repos with many stars though.

Reply:
will do, arkworks is a very popular Rust library for SNARKs which are a type of zero knowledge proofs, so it should figure somewhere.